spec/tests/honeybee_schedule_spec.rb

```diff
@@ -182,8 +192,8 @@
     expect(filename).to eq 'Solstice_Increasing.csv'
     columns = schedule_csv[:columns]
     expect(columns.size).to eq 2
-    expect(columns[0].size).to eq 8760*6 + 1
-    expect(columns[1].size).to eq 8760*6 + 1
+    expect(columns[0].size).to eq 8784*6 + 1
+    expect(columns[1].size).to eq 8784*6 + 1
```

Comment:
Hm. Come to think of it, why did we pick a leap year? This is definitely the rarer case. I'll change this back once we set the default calendar year to 2017.